wp-content/themes/core/pcss/base/media/_images.pcss

```diff
@@ -85,13 +86,37 @@ img {
 .alignleft {
 	@media (--viewport-medium) {
 		float: left;
-		margin-right: var(--spacer-lg);
+		margin-right: var(--spacer-4);
 	}
 }
 
 .alignright {
 	@media (--viewport-medium) {
 		float: right;
-		margin-left: var(--spacer-lg);
+		margin-left: var(--spacer-4);
+	}
+}
+
+.aligncenter {
+
+	.t-content & {
+		width: 100%;
+		margin-left: var(--grid-gutter-small-negative);
+		margin-right: var(--grid-gutter-small-negative);
+
+		@media (--viewport-medium) {
+			margin-left: auto;
+			margin-right: auto;
+		}
+	}
+
+	.t-content &.wp-image--caption .wp-caption-text {
+		margin-left: var(--grid-gutter-small);
```

Comment:
I'd go with padding for the mobile implementation, less funky than margins can sometimes be with the box model ;)

Reply:
@ryanurban using padding here would extend the border bottom to the edge, which won't match design. Is it worth it to add the bottom border as a pseudo element or just stick with margins here?

Reply:
Ah I see, carry on!